examples/hackweek_demo/run_demo.py

```diff
@@ -112,7 +164,7 @@ def get_user_lookup() -> pd.DataFrame:
     return options_map.reset_index()
 
 
-def username_to_address(lookup: pd.DataFrame, selected_list: pd.DataFrame) -> pd.Series:
+def username_to_address(lookup: pd.DataFrame, selected_list: pd.Series) -> pd.Series:
     """Helper function to lookup selected users/addrs to all addresses
```

Comment:
```suggestion
    """Look up selected users/addrs to all addresses.
```

lest we not forget the word of PEP 257:
> The docstring is a phrase **ending in a period.** It prescribes the function or method’s effect **as a command** (“Do this”, “Return that”), **not as a description;** e.g. don’t write “Returns the pathname …”.